extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -110,3 +110,92 @@
+  background-repeat: no-repeat !important;
+  background-position: center !important;
+  background-size: 0.25em !important;
+  background-color: white !important;
+  border-radius: 50% !important;
+  width: 1.25em !important;
+  height: 1.25em !important;
+  pointer-events: none !important;
+}
+
+.i-amphtml-story-interactive-disclaimer-bubble {
+  overflow: hidden !important;
+  padding: 1em !important;
+  text-align: initial !important;
+  box-sizing: border-box !important;
+  font-size: 1em !important;
+  background: white;
+  border-radius: 1em;
+  box-shadow: 0px .2em .2em 0px #00000040 !important;
+  pointer-events: all !important;
+}
+
+.i-amphtml-story-interactive-disclaimer:not([active])
+  .i-amphtml-story-interactive-disclaimer-bubble {
+  display: none !important;
+}
+
+.i-amphtml-story-interactive-disclaimer-url {
+  color: #5f6368 !important;
+  padding-bottom: 0.2em !important;
+}
+
+.i-amphtml-story-interactive-disclaimer-link {
+  color: #005af0 !important;
+  font-weight: bold !important;
+  text-decoration: none !important;
+}
+
+.i-amphtml-story-interactive-disclaimer-close {
+  position: absolute !important;
+  bottom: 0 !important;
+  right: 0 !important;
+  cursor: pointer !important;
+  background: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="#9AA0A6"><rect x="0.787868" y="1.83337" width="1.47851" height="13.2636" rx="0.739255" transform="rotate(-45 0.787868 1.83337)"/><rect x="10.1669" y="0.787868" width="1.47851" height="13.2636" rx="0.739255" transform="rotate(45 10.1669 0.787868)"/></svg>') !important;
+  width: 3em !important;
+  height: 3em !important;
```

Comment:
Adding `font-size: inherit;` will make this size responsive since a system default is set on buttons.